modules/java/generator/gen_java.py

```diff
@@ -1535,7 +1538,7 @@ def isSmartClass(self, classname):
         '''
         Check if class stores Ptr<T>* instead of T* in nativeObj field
         '''
-        return self.isWrapped(classname) and self.classes[classname].base
+        return False
```

Comment:
Maybe it will be better to revert line 1350 from previous PR and line 1541 in this PR (probably `self.classes[classname].base` check should be omitted).